packages/connector-puppeteer/tests/actions.ts

```diff
@@ -0,0 +1,122 @@
+// import { URL } from 'url';
+import * as path from 'path';
+
+import test from 'ava';
+// import * as sinon from 'sinon';
+
+// import { Server } from '@hint/utils-create-server';
+
+// import { test as testUtils } from '@hint/utils';
+import { Engine, Events } from 'hint';
+
+import Connector from '../src/connector';
+// import { validEvent } from './_valid-event';
+
+// const { generateHTMLPage } = testUtils;
+const name = 'puppeteer';
+
+test(`[${name}] User actions throw an exception if action is not found`, (t) => {
+
+    t.plan(1);
+
+    const engine: Engine<Events> = {
+        emit(): boolean {
+            return false;
+        },
+        async emitAsync(): Promise<any> { },
+        on(): Engine {
+            return null as any;
+        },
+        timeout: 10000
+    } as any;
+
+    try {
+
+        const connector = new Connector(engine, {
+            actions: [{
+                file: `fake-file`,
+                on: 'beforeTargetNavigation'
+            }],
+            detached: true
+        });
+
+        connector.close();
+    } catch (e) {
+        t.true((e.message as string).startsWith(`Couldn't load user action in "`));
+    }
+});
+
+test(`[${name}] User action gets loaded and throws if it does not have the right shape`, (t) => {
+
+    t.plan(1);
+
+    const actionPath = path.join(__dirname, './fixtures/invalid-action.js');
+
+    const engine: Engine<Events> = {
+        emit(): boolean {
+            return false;
+        },
+        async emitAsync(): Promise<any> { },
+        on(): Engine {
+            return null as any;
+        },
+        timeout: 10000
+    } as any;
+
+    try {
+
+        const connector = new Connector(engine, {
+            actions: [{
+                file: actionPath,
+                on: 'beforeTargetNavigation'
+            },
+            {
+                file: actionPath,
+                on: 'afterTargetNavigation'
+            }],
+            detached: true
+        });
+
+        connector.close();
+    } catch (e) {
+        t.is(e.message, `User action "${actionPath}" doesn't export a member "action".`);
+    }
+});
+
+
+test(`[${name}] User action gets loaded if it does not have the right shape`, (t) => {
```

Comment:
I think the description is wrong, it should be `[${name}] User action gets loaded if it has the right shape`

Reply:
I've updated the titles of the tests. Hopefully they make more sense now and are correct :)